### PR TITLE
added api call for fetching event actions history

### PR DESCRIPTION
**Pull request by jenjenp**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -1753,13 +1753,47 @@ paths:
...
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/events/{event_id}/actions:
+    get:
+      tags:
+        - Events
+      summary: "Fetch event actions"
+      description: "Fetch event actions"
+      parameters:
+          - in: path
```
Comment (chook) [inline services.yaml]: please make sure your "tab" are 2 characters and not 4

File: services.yaml [hunk 2]
```diff
@@ -1753,13 +1753,47 @@ paths:
...
+            name: event_id
+            description: Event to use
+            type: string
+            required: true
+          - in: query
+            name: type
+            description: Filter by specific type
+            type: string
+            required: false
```
Comment (chook) [inline services.yaml]: no need to put required:false. if it's not required simply remove the attribute altogether

File: services.yaml [hunk 3]
```diff
@@ -1753,13 +1753,47 @@ paths:
...
+      responses:
+        200:
+          description: Event actions
+          schema:
+            actions:
+              type: "array"
+              items:
+                $ref: "#/definitions/EventAction"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
```
Comment (chook) [inline services.yaml]: please merge the volumetrics as it contains a lot of the fixes we need such as having errors as "#/responses/.." and not definitions

**Review by chook — CHANGES_REQUESTED** [review]

File: services.yaml (hunk 1 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 2 above)
Comment (jenjenp) [reply services.yaml]: fixed

**Comment by chook** [comment]
@jenjenp resolve conflicts

**Review by chook — APPROVED** [review]

**Pull request merged**